### ISSUE TITLE
Hindi - Extended (Simplified)

### ISSUE THREAD

**Issue by NripPrajapat** [dessalines/thumb-key#1132]
### Language

Hindi

### Variant

Extended (with Indo-Arabic Numbers)

### Kind of request

New Layout

### More details

It is a variant and simplified, more subtle layout for Hindi Language.

### Acknowledgements

- [X] I have written a short but informative title.
- [X] I have searched the existing layouts and issues to confirm that this  layout either does not exist or was not requested already
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements. 🤡
- [X] I have filled out all of the requested information in this form.

**Comment by NripPrajapat** [comment]
[Hi.txt](https://github.com/user-attachments/files/17598467/Hi.txt)
It is the second and easiest variant.
Check if is written right, compile and test is first. Thanks.

**Comment by NripPrajapat** [comment]
There was a simple mistake, I fixed it and it's great now, please update it. To following file

[Hi.txt](https://github.com/user-attachments/files/17621951/Hi.txt)


It would be great, if it stops auto capitalisation, because there is no capitalisation in Hindi language.
It should change if one clicks capitalisation, because I used it to write indian numbers.

**Comment by dessalines** [comment]
Now that the file is added, you can click on edit and do this yourself. Watch a video on how to use github if you don't know how to do this.

**Comment by dessalines** [comment]
You should create a custom hindi numeric keyboard, and remove the shifted keyboard then, because its specifically for capitalization.

**Comment by NripPrajapat** [comment]
How should I make numeric keyboard, can you please the Base for that?

**Comment by dessalines** [comment]
Check out https://github.com/dessalines/thumb-key/blob/main/app/src/main/java/com/dessalines/thumbkey/keyboards/Numeric.kt for an example.

**Comment by NripPrajapat** [comment]
Hi extended without shifted keys
[Hi extended without shifted keys.txt](https://github.com/user-attachments/files/17634351/Hi.extended.without.shifted.keys.txt)

Numpad Hindi
[Hindi numpad.txt](https://github.com/user-attachments/files/17634363/Hindi.numpad.txt)

**Comment by NripPrajapat** [comment]
I tried learning how to use github, but it's not that simple, so I helped with whatever I could, anyway I played my part  i dont think i could help any further, thanks for co-operating with me, thanks for helping me out, if in future I think to add something up, I guess I will try to learn then. Thanks for your efforts. Peace.

**Comment by NripPrajapat** [comment]
> Thx, it'll probably be next week until I get to it. So this is a 3rd keyboard now? Or should it affect both the ones you already made.

You didn't included this in the update?

**Comment by NripPrajapat** [comment]
> Thx, it'll probably be next week until I get to it. So this is a 3rd keyboard now? Or should it affect both the ones you already made.

This is actually to replace extended version (V2), not 3rd.

**Comment by dessalines** [comment]
Then you'll need to edit that keyboard yourself. Watch a video on how to do a github pull request if you can't figure this out.

**Comment by NripPrajapat** [comment]
> Then you'll need to edit that keyboard yourself. Watch a video on how to do a github pull request if you can't figure this out.

Do it for the last time, because I really tried and still it's going over my head, BTW I dont have android studio or Computer, so I request you to do it for sake of me and our HINDI writing friends.

Regards.

**Comment by dessalines** [comment]
No. You do not need anything but a text editor. Find someone who can do this for you.
